app/models/article.rb

```diff
@@ -446,12 +458,7 @@ def before_destroy_actions
 
   def evaluate_front_matter(front_matter)
     self.title = front_matter["title"] if front_matter["title"].present?
-    if front_matter["tags"].present?
-      self.tag_list = [] # overwrite any existing tag with those from the front matter
-      tag_list.add(front_matter["tags"], parser: ActsAsTaggableOn::TagParser)
-      remove_tag_adjustments_from_tag_list
-      add_tag_adjustments_to_tag_list
```

Comment:
Removed these because they are redundant. This will be handled with `validate_tag`